firebase/rules/tests/servers.spec.ts

```diff
@@ -0,0 +1,33 @@
+import 'jest';
+import * as matchers from '../matchers';
+import * as targaryen from 'targaryen';
+
+import { SYSTEM_USER } from '@machinelabs/models';
+
+const anonymousUser = targaryen.util.users.anonymous;
+const rules = require('../database.rules.json');
+
+describe('/servers', () => {
+  const currentUser = { uid: '1', email: 'foo@bar.com', provider: 'github' };
+  const systemUser = { uid: SYSTEM_USER };
+
+  beforeEach(() => {
+    jest.addMatchers(matchers);
+
+    targaryen.util.setFirebaseData();
+
+    targaryen.util.setFirebaseRules(rules);
+  });
+
+  it('Only System-user can read into the servers node', () => {
+    expect(anonymousUser).cannotRead('/servers');
+    expect(currentUser).cannotRead('/servers');
+    expect(systemUser).canRead('/servers');
+  });
+
+  it('No users can write into the servers node', () => {
+    expect(anonymousUser).cannotWrite('/servers', '');
+    expect(currentUser).cannotWrite('/servers', '');
+    expect(systemUser).cannotWrite('/servers', '');
+  });
```

Comment:
Same here. Let's make it a single spec.

Reply:
Any preference how to name this spec file?

Reply:
The name of the file is fine, but the spec can be merged. But the more I think about it, the more i"m okay with leaving it as is.

Reply:
Personally I don't mind having them in separate files. I can understand the point of wanting them merged into one file, because they test all the same, but still, it's a different URL, so .. personally I would keep them in separated files if it's not that big of an issue.

Reply:
Ah I'm sorry. I didn't notice those where two different files in the first place!

Nevermind then!